[2025-04-18T00:58:13.485+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T00:58:13.560+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_new_orders.dbt_test manual__2025-04-18T00:57:33.796756+00:00 [queued]>
[2025-04-18T00:58:13.574+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_new_orders.dbt_test manual__2025-04-18T00:57:33.796756+00:00 [queued]>
[2025-04-18T00:58:13.576+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-18T00:58:13.593+0000] {taskinstance.py:2890} INFO - Executing <Task(TriggerDagRunOperator): dbt_test> on 2025-04-18 00:57:33.796756+00:00
[2025-04-18T00:58:13.602+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'load_new_orders', 'dbt_test', 'manual__2025-04-18T00:57:33.796756+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/store_to_sql_orders.py', '--cfg-path', '/tmp/tmpmdb9jczt']
[2025-04-18T00:58:13.606+0000] {standard_task_runner.py:105} INFO - Job 74: Subtask dbt_test
[2025-04-18T00:58:13.606+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3067) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-18T00:58:13.608+0000] {standard_task_runner.py:72} INFO - Started process 3074 to run task
[2025-04-18T00:58:13.861+0000] {task_command.py:467} INFO - Running <TaskInstance: load_new_orders.dbt_test manual__2025-04-18T00:57:33.796756+00:00 [running]> on host f693ba064550
[2025-04-18T00:58:14.147+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='azizjon' AIRFLOW_CTX_DAG_ID='load_new_orders' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T00:57:33.796756+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T00:57:33.796756+00:00'
[2025-04-18T00:58:14.149+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T00:58:14.583+0000] {trigger_dagrun.py:257} INFO - Waiting for dbt_etl_pipeline_test on 2025-04-18 00:57:33.796756+00:00 to become allowed state ['success'] ...
[2025-04-18T00:58:19.667+0000] {trigger_dagrun.py:271} INFO - dbt_etl_pipeline_test finished with allowed state success
[2025-04-18T00:58:19.739+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T00:58:19.740+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=load_new_orders, task_id=dbt_test, run_id=manual__2025-04-18T00:57:33.796756+00:00, execution_date=20250418T005733, start_date=20250418T005813, end_date=20250418T005819
[2025-04-18T00:58:19.958+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T00:58:20.026+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T00:58:20.039+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
